packages/apollo/src/Command.ts

```diff
@@ -103,6 +103,13 @@ export abstract class ProjectCommand extends Command {
     this.ctx = { flags, args } as any;
 
     const config = await this.createConfig(flags);
+
+    if (!config) {
```

Comment:
Configs can now be null :)

Reply:
I use `this.error` and `this.exit` here because there aren't many errors that happen in the CLI itself. Most happen in the language server. 

Errors in the CLI already have a handler (this.error) and should return with an exit code of 1. We _could_ wrap this up into another function but I don't see the need yet.